NextCenturyCorporation/itm-evaluation-dashboard
# ITM 363 password reset

Added CACI SMTP and corrected reset link and text.
The code is on production now for testing.
To test:
1. go to the UI: https://darpaitm.caci.com/
2. if you're logged in then log out
3. Click on the `Forgot Password ` link: https://darpaitm.caci.com/login#forgot
4. Type in the email that your account is associated with
5. Check your email, and go through the reset password process
6. Login with new password
